crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -188,6 +201,44 @@ impl CallParachainCommand {
 		}
 	}
 
+	// Sends an extrinsic to the chain using the call data.
+	async fn send_extrinsic_from_call_data(
+		&mut self,
+		api: &OnlineClient<SubstrateConfig>,
+		call_data: &str,
+		cli: &mut impl cli::traits::Cli,
+	) -> Result<()> {
+		// Resolve who is signing the extrinsic.
+		let suri = match self.clone().suri {
+			Some(suri) => suri,
+			None => cli.input("Signer of the extrinsic:").default_input(DEFAULT_URI).interact()?,
+		};
+		cli.info(format!("Encoded call data: {}", call_data))?;
+		if !self.skip_confirm &&
+			!cli.confirm("Do you want to submit the extrinsic?")
+				.initial_value(true)
+				.interact()?
+		{
+			display_message(
+				&format!(
+					"Extrinsic with call data {call_data} was not submitted. Operation canceled by the user."
+				),
+				false,
+				cli,
+			)?;
+			return Ok(());
+		}
+		let spinner = cliclack::spinner();
+		spinner.start("Signing and submitting the extrinsic, please wait...");
+		let result = sign_and_submit_extrinsic_with_call_data(api.clone(), call_data, &suri)
+			.await
+			.map_err(|err| anyhow!("{}", format!("{err:?}")))?;
+
+		spinner.stop(format!("Extrinsic submitted successfully with hash: {:?}", result));
+		display_message("Parachain calling complete.", true, cli)?;
```

Comment:
I think we should avoid using 'parachain' as the narrative is now moving to 'rollups'. Suggest we also add aliases to the command, but that can be done later. If parachain is littered everywhere, it will just make for more work in the future.

Perhaps something generic and simple like 'Call complete.'